test/fast/Unit tests/nvm_default_packages

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+die () { echo "$@" ; cleanup ; exit 1; }
+cleanup () {
+  rm -rf "$(nvm_version_path v6.10.1)" ../../../default-packages
+}
+
+cat > ../../../default-packages << EOF
+gulp
+bower@1.7.9
+
+# commented-package
+
+webpack/webpack
+EOF
+
+\. ../../../nvm.sh
+
+OUTPUT="$(nvm_ensure_version_installed 6.10.1)"
+EXIT_CODE=$?
+[ "_$OUTPUT" = "_" ] || die "expected 'nvm_ensure_version_installed 6.10.1' to have no output, got $OUTPUT"
+[ "_$EXIT_CODE" = "_0" ] || die "expected 'nvm_ensure_version_installed 6.10.1' to exit with 0, got $EXIT_CODE"
+
+OUTPUT="$(nvm exec 6.10.1 npm list -g | grep gulp)"
+[ "_$OUTPUT" = "_gulp" ] || die "expected 'npm list -g | grep gulp' to contain 'gulp', got $OUTPUT"
+
+cleanup
+
+cat > ../../../default-packages << EOF
+not-a-package-name
+EOF
+
+OUTPUT="$(nvm exec 6.10.1 npm list -g | grep gulp)"
```

Comment:
`npm ls -g` can take a `--depth=0` argument which should help the test be faster and more accurate